tests/features/targets/performance.feature

```diff
@@ -1,7 +1,5 @@
 Feature: Performance and Consistence adding and removing targets
-    As an admin,
-    Admin has deployed and bootstrapped RSTUF
-    Admin has issued a Token with scopes 'write:targets', 'delete:targets' and "read:tasks"
+    User has deployed and bootstrapped RSTUF
 
     Background:
         Given the API requester has access to RSTUF API
```

Comment:
You need to remove it, the FT is failing because is not finding it
https://github.com/repository-service-tuf/repository-service-tuf/pull/501/checks#step:4:1030